src/main/java/in/pune/royforge/eledgerapi/data/repo/WalletEntityRepository.java

```diff
@@ -7,6 +7,8 @@
 
 public interface WalletEntityRepository extends PagingAndSortingRepository<WalletEntity, Long> {
 
+	Iterable<WalletEntity> findByLenderId(String lenderId);
```

Comment:
change return type from Iterable<WalletEntity> to List<WalletEntity> and then use it.

Reply:
Done